CypressTests/cypress/support/payloads/Transfers/AuthorisedUserCanCreateNewMATTransfer.spec.js

```diff
@@ -0,0 +1,14 @@
+
+const transferringAcademyUkprnAAndOutgoingTrustUkprn = 10066875
```

Comment:
These values seem to be consistent across a few files, are they valid in all envs (i.e. both dev and test), and might they be better defined in a general constants config file instead?

Reply:
Yes quite possibly.  Repeating myself which isn't GPP.  very good point.